azure/functions/decorators/blob.py

```diff
@@ -0,0 +1,50 @@
+from typing import Optional

```

Comment:
Change the module name to _blob - potential conflict possibility.

Reply:
As discussed, keeping the module name as blob since it is not exposed and wont cause a conflict.